### PR TITLE
Added Parameter to LaunchBrowser to set path

### PR DESCRIPTION
**Pull request by Rotario**
Hi!
This PR just adds a parameter to LaunchBrowser that sets the path to open.
I was distributing a small webapp to a few colleagues and it was a little bit jarring for them to be presented with the listing page and have to click "index.html".
Hope it's up to snuff, I didn't know if there was a way to allow LaunchBrowser() and LauncBrowser(pathname:string) to be called

### CODE REVIEW

**Review by jart — APPROVED** [review]
Looks good. Could you send an email assigning copyright per https://github.com/jart/cosmopolitan/blob/master/CONTRIBUTING.md ?

**Comment by jart** [comment]
Ping. Are you still interested in contributing this change?

**Comment by Rotario** [comment]
Hey, I've already contributed before! And assigned copyright!
Thanks,
Rowan

**Comment by Rotario** [comment]
So yeah, please!

File: tool/net/redbean.c [hunk 1]
```diff
@@ -3514,7 +3518,8 @@ static void LuaSetIntField(lua_State *L, const char *k, lua_Integer v) {
 }
 
 static int LuaLaunchBrowser(lua_State *L) {
-  LaunchBrowser();
+  const char *p = luaL_checkstring(L, 1);
```
Comment (jart) [inline tool/net/redbean.c:3521]: Consider `luaL_optstring(L, 1, "/")`

**Review by jart — CHANGES_REQUESTED** [review]
My mistake. Thanks for your contribution! One small recommended change to make parameter optional.

File: tool/net/redbean.c (hunk 1 above)
Comment (Rotario) [reply tool/net/redbean.c:3521]: Ah cool - I was searching for "lua optional parameters" in the docs but couldn't find anything.
PS: Ah reading more, Lua's like JS in that all parameters are optional - my bad!

**Comment by Rotario** [comment]
I've updated the Lua binding to use optstring

**Review by jart — APPROVED** [review]
Thanks!

**Comment by jart** [comment]
New release available for you! https://justine.lol/redbean/index.html

**Pull request merged**